packages/salesforce-adapter/src/filters/custom_object_instances_references.ts

```diff
@@ -82,10 +80,42 @@ const createWarnings = async (
   instancesWithEmptyIds: InstanceElement[],
   missingRefs: MissingRef[],
   illegalRefSources: Set<string>,
-  customObjectPrefixKeyMap: Record<string, string>,
   dataManagement: DataManagement,
   baseUrl?: string,
 ): Promise<SaltoError[]> => {
+  const createOmittedInstancesWarning = async (
+    originTypeName: string,
+    missingRefsFromOriginType: MissingRef[],
+  ): Promise<SaltoError> => {
+    const typesOfMissingRefsTargets: string[] = _(missingRefsFromOriginType)
+      .map(missingRef => missingRef.origin.field?.annotations[FIELD_ANNOTATIONS.REFERENCE_TO])
+      .filter(isDefined)
+      .uniq()
+      .value()
```

Comment:
```suggestion
    const typesOfMissingRefsTargets: string[] = _.uniq(missingRefsFromOriginType
      .map(missingRef => missingRef.origin.field?.annotations[FIELD_ANNOTATIONS.REFERENCE_TO])
      .filter(isDefined))
```
Nit: Not a fan of the **lodash iterable**, consider the following instead